tap_salesforce/client.py

```diff
@@ -224,24 +278,26 @@ def get_records(
                 yield from self.get_records(
                     table,
                     fields,
-                    replication_key,
                     start_date=start_date + timedelta(seconds=i * nth),
                     end_date=start_date + timedelta(seconds=((i + 1) * nth)),
                     limit=limit,
                     shrink_window_factor=shrink_window_factor + 1,
                 )
 
     def _paginate(
-        self, method: str, path: str, data: Dict = None, params: Dict = None
+        self,
+        method: str,
+        path: str,
+        data: Dict = None,
+        params: Dict = None,
     ) -> Generator[Dict, None, None]:
         next_page: Optional[str] = path
         while True:
             resp = self._make_request(method, next_page, data=data, params=params)
 
             resp_data = resp.json()
-
-            yield from resp_data.get("records", [])
-
+            for record in resp_data.get("records", []):
+                yield record
```

Comment:
Why the change here?

Reply:
you are right, I don't have to change here.